### ISSUE TITLE
AdaSpeech images

### ISSUE THREAD

**Issue by erogol** [erogol/TTS-papers#16]
<img width="278" alt="Linear Layer" src="https://user-images.githubusercontent.com/1402048/135750827-24f74e2e-ec2f-4af7-ac6f-618309d7178d.png">

<img width="939" alt="d) Phoneme level" src="https://user-images.githubusercontent.com/1402048/135750837-3958d283-040e-4959-891b-10633455c7b4.png">

<img width="352" alt="Linear Layer" src="https://user-images.githubusercontent.com/1402048/135750845-c606e191-9f6e-4dc0-b50b-b3a1b3bc79c9.png">

<img width="977" alt="Algorithm 1 Pre-training, fine-tuning and inference of AdaSpeech" src="https://user-images.githubusercontent.com/1402048/135750852-9e3291ae-8d37-41fc-9c88-7daccd2a2104.png">

<img width="962" alt="Metric Setting" src="https://user-images.githubusercontent.com/1402048/135750857-21565d3d-ef8a-42db-b3a6-ca225457fa1c.png">

<img width="436" alt="Setting" src="https://user-images.githubusercontent.com/1402048/135750860-2c8e610b-c9b3-4bd7-a94f-5479ac6c87b6.png">
